src/api/script/api_test_in_spec.sh

```diff
@@ -63,15 +63,12 @@ EOF
 /usr/sbin/memcached $MEMCACHED_USER -l 127.0.0.1 -d -P $MEMCACHED_PID_FILE || exit 1
 
 # migration test
```

Comment:
I understand why the cleanup is needed. But as I said, this is also done in the `001` migration. So you think that there will be more differences and we should test this?